wiki/Beatmap_Editor/Song_Setup/en.md

```diff
@@ -1,222 +1,258 @@
 
-<!-- -->
+#### Beatmap Creator
 
--   **Difficulty:** **The name of the difficulty of map.** The dropdown contains the preset names: Easy, Normal, Hard, Insane. You can select one of the default names, or create your own name. While it's good to be creative, try to make it very clear which one is harder than others as ambiguous difficulty names can annoy players. This would also be the field where you indicate a guest mapper, if this is their difficulty (e.g. "Larto's Hard").
+**The creator's name that created the beatmap.** The game client should handle this automatically and will not allow you to edit this field, that is to say, if you are the creator.
 
-<!-- -->
+*Notice: The names of [guest creators](/wiki/GD) belong to the [difficulty](#difficulty) and/or [tags](#tags) fields.*
 
--   **Source:** **Where this song originate from.** This field should generally only be included for Anime or Video Game songs, and occasionally for Novelty (Movies, TV, or Internet) songs. If the song is from an anime or video game, or is famous for its use as a TV or movie theme, the title of that source goes here. Rock and Pop songs should generally leave the field blank. **This is not to be used for things like album titles.**
+#### Difficulty
 
-<!-- -->
+**The name of the beatmap, in other words, the difficulty name.** The dropdown contains the preset names of `Easy`, `Normal`, `Hard`, and `Insane`. The creator can select one of the preset names from the dropdown menu or be creative use their own. While it is good to be creative, do try to make the difficulty names very clear as to which difficulty they represent. Ambiguous difficulty names will annoy players and is unrankable.
 
--   **Tags:** **Keywords to help in searches.** Common uses include album names, guest difficulty creators, company name and other terms that may be used by people when searching for specific wants (like eroge or visual novel for example if your beatmap's song is based on an eroge visual novel). If your map has "collab" elements mapped by other mappers, their names can go here. Tags are separated by spaces. It is not compulsory to fill in but is greatly appreciated when using the Web search and osu! search in-game.
-    -   Note: If you have a tag of "qwertyuiop" as a tag on your beatmap, you can also find your beatmap in-game by typing in "qwertyuiop".
+This field can also be used to indicate that it is a guest difficulty (e.g. "Larto's Hard").
 
-Again, to avoid issues, please make sure that everything (aside from Difficulty name) is filled out identically for each difficulty.
+#### Source
 
-Difficulty
-----------
+**The origin of the song.** Generally, this field should be used only for anime or video game songs and occasionally for novelty (movies, TV, or internet) songs. If the song is from an anime, a video game, or is famous for its use in a TV or movie theme; then the title of that source should be added in this field. If otherwise, this field should be left empty.
```

Comment:
If the song is from an anime, a video game, or is famous for its use in a TV programme or a movie; then the title of that source should be added in this field. 
maybe?